APItokenmanager/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Assuming you already have a Dockerfile setup for Django
+FROM python:3.8
+
+# Set the working directory
+WORKDIR /app
+
+# Copy project code
+COPY . /app
+
+# Install dependencies
+RUN pip install --upgrade pip
+RUN pip install -r requirements.txt
+
+# Copy the .env file to the container
+COPY .env /app/.env
+
+# Install python-decouple to read environment variables
+RUN pip install python-decouple
+
+# Run migrations
+RUN python manage.py makemigrations
+RUN python manage.py migrate
```

Comment:
Combine into single clause to limit layers in your docker image

Reply:
I have updated the docker file to combine into single clause as suggested